.github/workflows/merge-gate.yml

```diff
@@ -45,14 +45,20 @@ jobs:
     permissions:
       contents: read
 
+  performance-checks:
```

Comment:
to comment out the performance-checks as suspect need another iteration on code & link to new issue